upgrade-guide.md

```diff
@@ -10,9 +10,11 @@ TODO
 
 - **Model copy**: mappings between model components have been modified to open design to arbitrary collections. It means that you can add additional information to these mappings for your own copy functions.
 
+- **Mesh loading**: mesh loading functions are now returning the mesh loaded from the filename.
+
 **How to upgrade**
```

Comment:
Is there one "How to upgrade" gathering all items or one "How to upgrade for each?